config/locales/en.yml

```diff
@@ -291,6 +291,7 @@ en:
   multifactor_auths:
     recovery_code_html: 'You can use a valid  <a href="https://guides.rubygems.org/setting-up-multifactor-authentication/#using-recovery-codes-and-re-setup-a-previously-enabled-mfa", class="t-link">recovery code</a> if you have lost access to your multi-factor authentication device or to your security device.'
     incorrect_otp: Your OTP code is incorrect.
+    session_expired: Your session has expired.
```

Comment:
Should this make reference to the OTP?

Reply:
The OTP itself isn't expiring, but rather the time spent on the page. I updated the text to make it more granular - specified the login page session is what expired.